### PR TITLE
netdata: replace notification with placeholder

### PR DESCRIPTION
**Pull request by lasers**
This removes annoying notification about not finding `nic`.

I don't use netdata but when I'm testing this module, I sometimes run into this annoying notification for many different reasons such as...
* Moving computer/laptop around
* Unplugging cords
* Not running `nm-applet` on startup
* Use `Disconnect` in `nm-applet`
* Whatever the reason may be...

This is quite annoying so I think we should remove the notification part. I'm not sure what stance we would like to take for the _New!_ `nic` placeholder so I included it in the `format` because all other placeholders are in the format too.... with `degraded` color.

Maybe we go with `bad` instead? One thing to point out. The alignment looks slightly off between `\?color=` versus `color threshold`.

`samecolor samecolor` OK.
`samecolor differentcolor` SLIGHTLY OFF.

I don't know if it's already possible, but if we don't like this new feature, we could try something like... `[\?color=degraded&if=nic&is=lo NIC: {nic}]` to get yellow `NIC: lo` appear only when it is `lo`, otherwise... nothing. Idk. 

EDIT: I removed `NIC: ` and use ~~`333`~~. No color for `nic`.

### CODE REVIEW

**Comment by tobes** [comment]
I think removing the notification is good.

adding `{nic}` is nice too 👍 

I think we should leave the format as it was.

**Comment by tobes** [comment]
actually the adding `{nic}` to the format has grown on me.

@ultrabug do you have a view on the new format?

**Comment by lasers** [comment]
@tobes 😄 😄 😄 😄 😄 **¥å¥å¥å¥å¥å¥å¥å¥å¥å¥å¥å¥å¥** I would like to use this as a new format... Here, try it for a while and see if it grows on you too.
`format = '{nic} [\?color=down Speed(Kb): {down}↓ {up}↑]`
`[\?color=total Traffic(Mb): {download}↓ {upload}↑ {total}↕]'`

I think that new format makes things a bit more readable... and eventually somebody polish this some more by removing locked width to help it stay small...  Does `LAN (KiB)` makes sense in this context for speed or am I missing something here?

I don't mind `KiB` for speed, but `GiB` for traffic instead of `MiB` so I'm guessing we (or somebody else) should take a page from `sysdata` by adding `speed_unit` + `traffic_unit` where we can decide `dynamic, KiB, MiB, GiB`.

If that happens, then the new format, to include new unit placeholders, could be this...
`Speed ({speed_unit}): ... Traffic ({traffic_unit}): ... `

Then we could ponder if we can make do without the arrows... to see if we can make it more readable (and/or compact things). Slash instead of format. Little tweak there and there... and there... eventually should make the module more decent than before. Something like...
`Speed: 13.5 KiB/2.2 KiB  Traffic: 7.3 GiB/1.5 GiB [8.8]`
`Speed (KiB): 13.5/2.2  Traffic (GiB): 7.3/1.5 [8.8]` <-- I think this looks sharp.
`Speed (Kb): 13.5/2.2  Traffic (Gb): 7.3/1.5 [8.8]`
`Speed: 13.5 Kb/2.2 Kb  Traffic: 7.3 Gb/1.5 Gb [8.8 Gb]` <-- Hard to parse, imho.

Ha. Sounds fun? Do we even want separator back? Maybe fake/broken separator to replace that... `¦` to help breaking it up versus real thing. I'm guessing `[\?color=separator&show ¦]` in there could do a trick... All in good time. 👍 Will rebase the branch.

**Comment by ultrabug** [comment]
I'm okay with nic, thanks a lot !
